docs/API.md

```diff
@@ -136,3 +136,59 @@ const raw = joinConfigConversionResults(summarizedConfiguration, originalConfigu
 
 const formatted = formatOutput("eslintrc.js", raw);
 ```
+
+## Standalone Comment Conversion API
+
+> ⚠ This area of code is still considered experimental.
+> Use at your own risk.
+> Please file an issue on GitHub if you'd like to see changes.
+
+The individual per-file conversion logic within `convertComments` is available as a standalone function:
+
+### `convertFileComments`
```

Comment:
@JamesHenry this is published under v2.0.0-beta4 now, if you want to try it out. Is this the kind of API you're looking for?

I'd recommend using both `ruleCommentsCache` (for a small perf boost) and `ruleEquivalents` (for rule accuracy per arguments).

Reply:
Ooh I missed this tip and just used your snippet in the description, I'll look at adding them in